src/query/query_executor.c

```diff
@@ -22773,6 +22773,39 @@ qexec_execute_build_columns (THREAD_ENTRY * thread_p, XASL_NODE * xasl, XASL_STA
 	    {
 	      db_make_string_by_const_str (out_values[idx_val], "auto_increment");
 	    }
+
+	  if (attrepr->on_update.default_expr_type != DB_DEFAULT_NONE)
```

Comment:
I don't think it is ok to concatenate to auto_increment field, should be concatenated to default value. Please make a function or use a function to generate complete default value string.

Reply:
I saw this is the result for mysql query SHOW COLUMNS t from basic:

Field | Type | Null | Key  | Default | Extra
-- | -- | -- | -- | -- | --
t | timestamp | YES |   | CURRENT_TIMESTAMP | on update CURRENT_TIMESTAMP

This is how current's branch result for show columns from t: 

Field | Type | Null | Key  | Default | Extra
-- | -- | -- | -- | -- | --
b|TIMESTAMP|YES|| CURRENT_TIMESTAMP|ON UPDATE CURRENT_TIMESTAMP

I thought it makes sense to do it like mySQL, I'll make it lowercase too